documentation/development-docs/design-guidelines/parameter-best-practices.md

```diff
@@ -32,6 +32,31 @@ From the [_Strongly Encouraged Development Guidelines_](https://docs.microsoft.c
 
 If you there is a separate nomenclature for the parameter name, or if you would like to shorten the name of the parameter so it's easier to remember, you can add an alias attribute to your parameter to allow for this functionality.
 
+### Parameter Syntax
+
+In PowerShell documentation, brackets indicate optional. 
+Convention is as follows: 
+
+<command-name> 
+    -<Required Parameter Name> <Required Parameter Value>
+    [-<Optional Parameter Name> <Optional Parameter Value>]
+    [-<Optional Switch Parameters>]
+    [-<Optional Parameter Name>] <Required Parameter Value>
```

Comment:
Angle brackets `<>` are used for types. I find it ambiguous they are also used around parameter names.
How about:

```suggestion
command-name
    -RequiredParameterName <RequiredParameterValue>
    [-OptionalParameterName <OptionalParameterValue>]
    [-OptionalSwitchParameters]
    [-OptionalParameterName] <RequiredParameterValue>
```